miso-web/src/main/resources/internal/miso.properties

```diff
@@ -59,6 +59,9 @@ miso.notification.interop.enabled:false
 miso.notification.server.host:your.notification.server
 miso.notification.server.port:8787
 
+# A white-space delimited list of run scanner instances to poll
+miso.runscanner.urls:http://localhost:8080/runscanner
```

Comment:
Done. Worth pointing out that 1) it causes non-critical failure 2) runscanner fails to deploy beside a MISO instance properly due to log4j weirdness, 3) the VisionMate is set to localhost.